update_ta_aliases_list.py

```diff
@@ -0,0 +1,53 @@
+# import json  # for testing purposes
+import argparse
+from ta_list_methods import get_all_ta_from_act, get_all_alias_facts_from_act, \
+    add_ta_to_map, add_ta_alias_to_map, create_config
+
+
+def parseargs():
+    """ Parse arguments """
+    parser = argparse.ArgumentParser(description='Updates list of threat actor \
+        aliases from an ACT platform')
+    parser.add_argument('--baseurl', required=True, help='URL to ACT platform \
+        installation')
+    parser.add_argument('--userid', required=True, help="User ID for ACT \
+        platform")
```

Comment:
As mentioned previously, I would consider to add this command line argument:

- output-json

In addition, also consider adding these 

- alias-filename
- new-alias-filename

And use these are arguements for create_config so it is possible to name the input and output alias config files by command line options in stead of changing code.